libs/template/templates/default-python/template/{{.project_name}}/src/{{.project_name}}/main.py.tmpl

```diff
@@ -1,16 +1,11 @@
-{{- /*
-We use pyspark.sql rather than DatabricksSession.builder.getOrCreate()
-for compatibility with older runtimes. With a new runtime, it's
-equivalent to DatabricksSession.builder.getOrCreate().
-*/ -}}
-from pyspark.sql import SparkSession
+from pyspark.sql import SparkSession, DataFrame
 
-def get_taxis():
-  spark = SparkSession.builder.getOrCreate()
+def get_taxis(spark: SparkSession) -> DataFrame:
   return spark.read.table("samples.nyctaxi.trips")
 
 def main():
-  get_taxis().show(5)
+  from databricks.sdk.runtime import spark
```

Comment:
In which environments does this work? Older runtimes, DLT, etc.?

Reply:
This works in DBR >= 12